src/python/WMCore/BossAir/Plugins/PyCondorPlugin.py

```diff
@@ -267,7 +267,7 @@ def __init__(self, config):
             self.proxy = self.setupMyProxy()
 
         # Build a request string
-        self.reqStr = "(Memory >= 1 && OpSys == \"LINUX\" ) && (Arch == \"INTEL\" || Arch == \"X86_64\") && stringListMember(GLIDEIN_CMSSite, DESIRED_Sites) && ((REQUIRED_OS==\"any\") || (GLIDEIN_REQUIRED_OS==REQUIRED_OS))"
```

Comment:
This is now automatically added by condor.  In fact, having `Memory >= 1` causes `condor_submit` to throw a warning and condor will ignore this.